acme/account.go

```diff
@@ -24,6 +24,7 @@ type Account struct {
 	PrivateKey         []byte
 	DomainsCertificate DomainsCertificates
 	ChallengeCerts     map[string]*ChallengeCert
+	HttpChallenge      map[string]map[string][]byte
```

Comment:
We really need to add an integration test to check that we support old acme.json.